analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/Extensions/IInvocationOperationExtensions.cs

```diff
@@ -42,4 +42,10 @@ internal static class IInvocationOperationExtensions
         || (invocation.TargetMethod.IsExtensionMethod
             && !invocation.Arguments.IsEmpty
             && state.ResolveCaptureAndUnwrapConversion(invocation.Arguments[0].ToArgument().Value).Kind == OperationKindEx.InstanceReference);
+
+    public static IOperation GetInstance(this IInvocationOperationWrapper invocation, ProgramState state) =>
```

Comment:
You could improve coverage by adding UTs for this extension method. This would be good practice in general. I'm sorry I missed mentioning this earlier.